60_networks.Rmd

```diff
@@ -0,0 +1,664 @@
+# Network learning and inference {#network-inference}
+```{r setup, echo=FALSE, results="asis"}
+library(rebook)
+chapterPreamble()
+```
+
+Estimating microbial association networks is a further common exploratory data analysis approach aimed at understanding the complex interplay of microbial communities in their natural habitat. Microbial networks consist of nodes, representing microbial species or taxa, and edges, expressing their association. The mathematical representation of a network is the adjacency matrix, which has a non-zero entry for each edge in the network. 
+
+A typical workflow for estimating a microbial network involves several analysis steps, including methods for zero handling, data normalization, and computing microbial associations. The inferred association matrix is then sparsified and transformed into an adjacency matrix. The resulting network can be analyzed using local network properties, such as centrality measures, or global measures that describe the overall structure of the network. Network plots provide further insight into the microbial community structure and allow for an exploratory analysis of the microbial relationships.
+
+Since microbial interactions are likely to change between conditions, e.g., between healthy individuals and patients, identifying network differences between groups is often an integral secondary analysis step. Differences can be detected visually by plotting the two networks side by side, or quantitatively using differential network analysis tools.
+
+All of the above steps are implemented in the R package NetCoMi [@peschel2021netcomi], which provides functionality for the complete workflow of constructing, analyzing, and comparing networks for microbiome data. In this section, we demonstrate the use of NetCoMi.
+
+
+## Intruduction to NetCoMi
+
+The package has a modular structure with a function for each step of the workflow. These are the main functions with their purpose and expected input:
+
+| Function               | Purpose                | Input
+|:----------------------:|:----------------------:|:----------------------:|
+| `netConstruct`       | Network construction   | Count matrix, phyloseq object, or association matrix|
```

Comment:
This refers to phyloseq but the OMA contents are not based on _phyloseq_ but instead of the alternative data container, _TreeSummarizedExperiment_. Hence this line may be confusing for new users who are not familiar with the differences. Does the `netConstruct` function support _TreeSummarizedExperiment_, or could this be added? We can provide help with this and it should be relatively straightforward to add support for TreeSE containers, too.